### DISCUSSION TITLE
Unexpected parameter: contributor_details Unexpected parameter: inlclude_entities

### DISCUSSION THREAD

**Discussion by vardhans07** [tweepy/tweepy#2092] [Questions] [answered]
when i try use tweepy using api-key but this error occured .how tackle this problem.

**Comment by vardhans07** [comment]
Unexpected parameter: contributor_details
Unexpected parameter: inlclude_entities

**Comment by Harmon758** [comment, marked as answer]
Like it says, whatever method/endpoint you're using doesn't expect those parameters that you're passing.

If you need further help, you'll need to provide the relevant code and what exactly you're trying to do.

**Reply by vardhans07** [reply]
import tweepy #to access the twitter api
from textblob import TextBlob
import pandas as pd
import numpy as np
import re
import matplotlib.pyplot as plt
#____________________________________________________________________________________________________
#We need to configure our file
config=pd.read_csv("./config.csv")
#____________________________________________________________________________________________________
# Importing the keys from Twitter api
twitterApiKey =config['twitterApiKey'][0]
twitterApiSecret = config['twitterApiSecret'][0]
twitterApiaccessToken = config['twitterApiaccessToken'][0]
twitterApiaccessTokenSecret =config['twitterApiaccessTokenSecre'][0]
#---------------------------------------------------------------------------------------------------
auth=tweepy.OAuthHandler(twitterApiKey,twitterApiSecret)
auth.set_access_token(twitterApiaccessToken, twitterApiaccessTokenSecret)
twitterApi = tweepy.API(auth,wait_on_rate_limit=True)
#---------------------------------------------------------------------------------------------------
# user Account use
twitterAccount='RajvardhanPatil'
#---------------------------------------------------------------------------------------------------
tweets = tweepy.Cursor(twitterApi.user_timeline,
                       screen_name=twitterAccount,
                       count=None,
                       since_id=None,
                       max_id=None,trim_user=True,exclude_replies=True,contributor_details=False,
                       inlclude_entities=False).items(5);
#----------------------------------------------------------------------------------------------------
df=pd.DataFrame(data=[tweet.text for tweet in tweets],columns=['Tweet'])
#print(df.head())

#Tis function used for clean unwanted data from tweet
def cleanupTweet (txt):
    txt= re.sub(r'@[A-Za-z0-9_]+', '', txt)
    txt= re.sub(r'#','',txt)
    txt = re.sub(r'RT:','',txt)
    txt = re.sub(r"https?:\/\/[A-za-z0-9\.\/]+'", '',txt, flags=re.MULTILINE)   
    return txt

df['Tweet']=df['Tweet'].apply(cleanupTweet)


def getTextSubjectivity(txt):
    return TextBlob(txt).sentiment.subjectivity

def getTextPolarity(txt):
    return TextBlob(txt).sentiment.polarity


df['Subjectivity']=df['Tweet'].applyt(getTextSubjectivity)
df['Polarity']=df['Tweet'].apply(getTextPolarity)
df=df.drop(df[df['Tweet']==''].index)

#print(df.head(50))

def getTextAnalysis(a):
    if a <0:
        return "Negative"
    elif a==0:
        return "Neutral"
    elif a>0:
        return "Positive"
    
df['score'] =df['Polarity'].apply(getTextAnalysis) 

print(df.head(5))

**Reply by vardhans07** [reply]
this my code try to run but 
Unexpected parameter: contributor_details
Unexpected parameter: inlclude_entities
Traceback (most recent call last):
  File "c:\Users\dell\Desktop\try_1\Using api key project tw-sentimetal analysis\new.py", line 31, in <module>
    df=pd.DataFrame(data=[tweet.text for tweet in tweets],columns=['Tweet'])
  File "c:\Users\dell\Desktop\try_1\Using api key project tw-sentimetal analysis\new.py", line 31, in <listcomp>
    df=pd.DataFrame(data=[tweet.text for tweet in tweets],columns=['Tweet'])
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\cursor.py", line 86, in __next__
    return self.next()
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\cursor.py", line 286, in next
    self.current_page = next(self.page_iterator)
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\cursor.py", line 86, in __next__
    return self.next()
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\cursor.py", line 167, in next
    data = self.method(max_id=self.max_id, parser=RawParser(), *self.args, **self.kwargs)
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\api.py", line 33, in wrapper
    return method(*args, **kwargs)
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\api.py", line 46, in wrapper
    return method(*args, **kwargs)
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\api.py", line 577, in user_timeline
    return self.request(
  File "C:\Users\dell\AppData\Local\Programs\Python\Python310\lib\site-packages\tweepy\api.py", line 265, in request
    raise Forbidden(resp)
tweepy.errors.Forbidden: 403 Forbidden
453 - You currently have Essential access which includes access to Twitter API v2 endpoints only. If you need access to this endpoint, you�ll need to apply for Elevated access via the Developer Portal. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve

**Reply by Harmon758** [reply]
Again, like the warnings indicate, those are not valid parameters for [`API.user_timeline`](https://tweepy.readthedocs.io/en/v4.13.0/api.html#tweepy.API.user_timeline) and the endpoint it uses.

As for your actual error, see https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728.

For code block usage, see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.

**Comment by vardhans07** [comment]
Ok,Thanks 🙏
